chat-plugins/scavengers.js

```diff
@@ -184,7 +185,9 @@ function formatQueue(queue = [], viewer, room) {
 	} else {
 		buffer = `<tr><td colspan=3>The scavenger queue is currently empty.</td></tr>`;
 	}
-	return `<div class="ladder"><table style="width: 100%"><tr><th>By</th><th>Questions</th></tr>${buffer}</table></div><table style="width: 100%"><tr><td style="text-align: left;">Auto Timer Duration: ${timerDuration} minutes</td><td>Auto Dequeue: <button class="button${!queueDisabled ? '" name="send" value="/scav disablequeue"' : ' disabled" style="font-weight:bold; color:#575757; font-weight:bold; background-color:#d3d3d3;"'}>OFF</button>&nbsp;<button class="button${queueDisabled ? '" name="send" value="/scav enablequeue"' : ' disabled" style="font-weight:bold; color:#575757; font-weight:bold; background-color:#d3d3d3;"'}>ON</button></td><td style="text-align: right;"><button class="button" name="send" value="/scav next 0">Start the next hunt</button></td></tr></table>`;
+	let template = `<div class="ladder"><table style="width: 100%"><tr><th>By</th><th>Questions</th></tr>${isStaff ? buffer : buffer.replace(/<button.*?>.+?<\/button>/gi, '')}</table></div>`;
```

Comment:
Wouldn't `.*` be better than `.+?`

Reply:
``.+?`` checks that there is something between the 2 button tags.  I chose to use lazy (the ``?``) cuz I don't want it matching the first <button> in the thing, until the last </button> and removing all the important details in between.

Looking back, there should be no difference between ``.+?`` and ``.*?`` since all buttons will have something in between.  Unless someone knows that there is some performance difference between the two, I think in this case, they could be used interchangeably as long as the lazy operator is there

Reply:
I'm not exactly the person to go to for even basic regexp so I'll ask someone to double check this.

Reply:
Should be fine